src/MasterSite.php

```diff
@@ -304,6 +304,56 @@ function ($tests) {
             }
         );
 
+        // Maximum revisions to keep whenever the editor save a post
+        add_filter(
+            'wp_revisions_to_keep',
+            function ($revisions) {
+                $revisions_to_keep = get_option('revisions_to_keep');
+                return $revisions_to_keep ?: $revisions;
+            },
+            10,
+            2
```

Comment:
It should be 1 as we have single argument(i.e. $revisions) pass to the function.